packages/fast-layouts-react/src/pane/pane.spec.tsx

```diff
@@ -0,0 +1,632 @@
+
+        expect(rendered.props().collapsed).toBe(false);
+    });
+
+    test("should set a `minWidth` inline style value equal to `collapsedWidth` if `collapsed` prop is passed", () => {
+        const rendered: any = shallow(
+            <Pane collapsed={true} collapsedWidth={300} managedClasses={managedClasses} />
+        );
+
+        expect(rendered.props().style.minWidth).toBe("300px");
+    });
+
+    test("should set a `minWidth` inline style value equal to `minWidth` prop if `collapsed` prop is false and `resizeable` prop is true", () => {
+        const rendered: any = shallow(
+            <Pane
+                collapsed={false}
+                minWidth={500}
+                collapsedWidth={300}
+                managedClasses={managedClasses}
+            />
+        );
+
+        expect(rendered.props().style.minWidth).toBe("500px");
+    });
+
+    test("should set a `minWidth` inline style value equal to `minWidth` prop if `collapsed` and `resizeable` props are false and the `width` prop or state is smaller than `midWidth`", () => {
+        const rendered: any = shallow(
+            <Pane
+                collapsed={false}
+                minWidth={500}
+                maxWidth={800}
+                managedClasses={managedClasses}
+            />
+        );
+
+        rendered.setProps({ width: 300 });
+
+        expect(rendered.props().style.minWidth).toBe("500px");
+
+        rendered.setState({ width: 301 });
+
+        expect(rendered.props().style.minWidth).toBe("500px");
+    });
+
+    test("should set a `minWidth` inline style value equal to `maxWidth` prop if `collapsed` and `resizeable` props are false and the `width` prop or state is larger than `maxWidth`", () => {
+        const rendered: any = shallow(
+            <Pane
+                collapsed={false}
+                minWidth={500}
+                maxWidth={800}
+                managedClasses={managedClasses}
+            />
+        );
+
+        rendered.setProps({ width: 900 });
+
+        expect(rendered.props().style.minWidth).toBe("800px");
+
+        rendered.setState({ width: 801 });
+
+        expect(rendered.props().style.minWidth).toBe("800px");
+    });
+
+    test("should set `collapsed` to value of `collapsed` prop when passed", () => {
+        const rendered: any = mount(
+            <Pane collapsed={true} managedClasses={managedClasses} />
+        );
+
+        expect(rendered.props().collapsed).toBe(true);
+    });
+
+    test("should default `overlay` to false if no `overlay` prop is passed", () => {
+        const rendered: any = mount(<Pane managedClasses={managedClasses} />);
+
+        expect(rendered.props().overlay).toBe(false);
+    });
+
+    test("should set `overlay` to value of `overlay` prop when passed", () => {
+        const rendered: any = mount(
+            <Pane overlay={true} managedClasses={managedClasses} />
+        );
+
+        expect(rendered.props().overlay).toBe(true);
+    });
```

Comment:
can we add another overlay test and check the styles.flexBasis has been set?

Reply:
Yeah I had this and then pulled it...I'll add it back in.